Makefile

```diff
@@ -34,8 +34,36 @@ ct-tls:
 		--suite eredis_tls_SUITE || { docker logs redis; exit 1; }
 	@docker rm -f redis
 
+# Generate and patch documentation.
+# The patching is needed to be able to generate documentation via Elixirs mix.
+# Following changes are needed:
+# - Handle links in headers, change '### <a name="link">Header</a> ###' to
```

Comment:
Minor: These are "link targets", not links.

Reply:
Updated